scripts/build

```diff
@@ -1,9 +1,29 @@
+#!/bin/bash
 set -e
 
 source $(dirname $0)/version
 
 cd $(dirname $0)/..
 
-mkdir -p bin
-[ "$(uname)" != "Darwin" ] && LINKFLAGS="-linkmode external -extldflags -static -s"
-CGO_ENABLED=0 go build -ldflags "-X main.VERSION=$VERSION $LINKFLAGS" -o bin/rancher-dns
+declare -A OS_ARCH_ARG
+
+OS_PLATFORM_ARG=(linux windows)
+OS_ARCH_ARG[linux]="amd64"
+OS_ARCH_ARG[windows]="amd64"
+
+if [ -n "$CROSS" ]; then
+    rm -rf bin/*
+    mkdir -p bin
+    for OS in ${OS_PLATFORM_ARG[@]}; do
+        for ARCH in ${OS_ARCH_ARG[${OS}]}; do
+            OUTPUT_BIN="bin/$OS/$ARCH/rancher-dns"
+            if test "$OS" = "windows"; then
+                OUTPUT_BIN="${OUTPUT_BIN}.exe"
+            fi
+            echo "Building binary for $OS/$ARCH..."
+            GOARCH=$ARCH GOOS=$OS CGO_ENABLED=0 go build \
```

Comment:
You should keep the linux build as same as it is. In here the LINKFLAG are missing and you don't a static build for linux build.

Reply:
Updated